operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -836,8 +769,8 @@ spec:
 
     ### Documentation
 
-    Documentation for the current release can be found on the Red Hat [Customer Portal](https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q2/html/using_amq_streams_on_openshift/index).
-    
+    Documentation for the current release can be found on the Red Hat [Customer Portal](https://access.redhat.com/documentation/en-us/red_hat_amq/2021.q4/html/using_amq_streams_on_openshift/index).
```

Comment:
```suggestion
    Documentation for the current release can be found on the Red Hat [Customer Portal](https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.0).
```

Reply:
The splash page isn't up yet, but I think we can use this link and not have to update again